### PR TITLE
Fix & enable 'ZeroUnit' scss-lint cop

### PR DESCRIPTION
**Pull request by joshlory**

### CODE REVIEW

File: dashboard/app/assets/stylesheets/application.css.scss [hunk 1]
```diff
@@ -2056,7 +2056,7 @@ $modal-no-icon-gap: 22px;
 
   .aniGif {
     display: block;
-    margin: 0px auto 6px;
+    margin: 0 auto 6px;
```
Comment (joshlory) [inline dashboard/app/assets/stylesheets/application.css.scss]: Need to figure out why `rake lint:scss` didn't catch this locally.

**Comment by ashercodeorg** [comment]
LGTM (other than hound failure)

**Comment by joshlory** [comment]
Fixed the HoundCI issue, which required updating `scss-lint` from 0.48 -> 0.49.  PTAL!

**Comment by ashercodeorg** [comment]
Excellent. LGTM!

**Pull request merged**